cli/args/import_map.rs

```diff
@@ -4,21 +4,21 @@ use deno_core::error::AnyError;
 use deno_core::serde_json;
 use deno_core::url::Url;
 
-use crate::file_fetcher::FileFetcher;
+use crate::file_fetcher::CliFileFetcher;
+use crate::file_fetcher::TextDecodedFile;
 
 pub async fn resolve_import_map_value_from_specifier(
   specifier: &Url,
-  file_fetcher: &FileFetcher,
+  file_fetcher: &CliFileFetcher,
 ) -> Result<serde_json::Value, AnyError> {
   if specifier.scheme() == "data" {
     let data_url_text =
       deno_graph::source::RawDataUrl::parse(specifier)?.decode()?;
     Ok(serde_json::from_str(&data_url_text)?)
   } else {
-    let file = file_fetcher
-      .fetch_bypass_permissions(specifier)
-      .await?
-      .into_text_decoded()?;
+    let file = TextDecodedFile::decode(
+      file_fetcher.fetch_bypass_permissions(specifier).await?,
+    )?;
```

Comment:
This is not as good as before (not left to right/top to bottom), but oh well. Not too bad...